lib/hanami/model/sql/types/schema/coercions.rb

```diff
@@ -192,6 +192,12 @@ def self.hash(arg)
                 raise ArgumentError.new("invalid value for Hash(): #{arg.inspect}")
               end
             end
+
+            def self.json(arg)
```

Comment:
It's called [here](https://github.com/hanami/model/pull/423/files#diff-92790045433e0619f41285e4ad4b4af0R34):

```
JSON     = Types::Strict::Nil | Types::Hash.constructor(Coercions.method(:json)) | Types::Array.constructor(Coercions.method(:json))
```

Without that constructors are raising errors because of wrong data type.